### ISSUE TITLE
Failed to handle ALLOCATE-REQUEST

### ISSUE THREAD

**Issue by NarimanMov** [pion/turn#35]
Hi, Thanks for your awesome code.

I have a problem. 
I've followed these commands to run the server
```
$ wget -q https://github.com/pions/turn/releases/download/1.0.3/simple-turn-linux-amd64
$ chmod +x simple-turn-linux-amd64
$ export USERS='user=password nariman=nariman'
$ export REALM=myDomain.com
$ export UDP_PORT=3478
$ ./simple-turn-linux-amd64
```

And I tried to check it out with [this tool](https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/) 
![photo_2019-03-06_13-17-59](https://user-images.githubusercontent.com/16634828/53872172-d47c0f80-4012-11e9-9a13-561e1faf90eb.jpg)

The terminal showing me this error
```
Failed to handle ALLOCATE-REQUEST from 69.175.34.181:61666: Relay already allocated for 5-TUPLE
Failed to handle ALLOCATE-REQUEST from 69.175.34.181:61668: Relay already allocated for 5-TUPLE
```
I didn't find any way to solve this.
I'm running this code on Google Cloud - Ubuntu 16.

**Comment by songjiayang** [comment]
same problem

**Comment by Sean-Der** [comment]
Hey everyone, sorry for the delay!

I believe this is solved by 06a2f68d47754b2195daf547a36b3a223aa2b641 before we didn't properly tear down allocations in some cases. I am going to be working on TURN everyday for a couple weeks, so if anyone has feedback I would love to hear!

Thanks for using Pion, and sorry for the delay.

**Comment by boratanrikulu** [comment]
Same here. Getting this error on Turn.
